apps/team-awesome-web/src/components/forms/select/Select.tsx

```diff
@@ -76,14 +82,18 @@ export const Select = ({ sortList, direction }: SelectProps) => {
   return (
     <>
       <Styled.Select name="key" onChange={onChange}>
-        {sortOptions[sortList].map((option) => {
-          return <Styled.Option value={option[1]}>{option[0]}</Styled.Option>;
-        })}
+        {sortOptions[sortList].map((option) => (
+          <Styled.Option key={option[1]} value={option[1]}>
```

Comment:
Using an object for your sort options would make your code a bit easier to read here, then you could use `option.label` and `option.value` instead of `option[0]` and `option[1]`.